specs/protocol/README.md

```diff
@@ -93,42 +106,80 @@ see [here](#signatures) for more details
+
+> [!IMPORTANT]
+> If `requiredPaymentDetails` is omitted, then any RFQs submitted for the respective offering must also omit `paymentDetails`. 
+
+#### `PayoutMethod`
+| field                     | data type                               | required | description                                                                                                                                         |
+| ------------------------- | --------------------------------------- | -------- | --------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `kind`                    | string                                  | Y        | Type of payment method (i.e. `DEBIT_CARD`, `BITCOIN_ADDRESS`, `SQUARE_PAY`). considered to be a unique identifier                                   |
+| `estimatedSettlementTime` | uint                                    | Y        | estimated time taken to settle an order. expressed in seconds                                                                                       |
+| `name`                    | string                                  | N        | Payment Method name. Expected to be rendered on screen.                                                                                             |
+| `description`             | string                                  | N        | Blurb containing helpful information about the payment method. Expected to be rendered on screen. e.g. "segwit addresses only"                      |
+| `group`                   | string                                  | N        | value that can be used to group specific payment methods together e.g. Mobile Money vs. Direct Bank Deposit                                         |
+| `requiredPaymentDetails`  | [JSON Schema](https://json-schema.org/) | N        | A JSON Schema containing the fields that need to be collected in the RFQ's selected payment methods in order to use this payment method.            |
+| `fee`                     | [`DecimalString`](#decimalstring)       | N        | Fee charged to use this payment method. absence of this field implies that there is no _additional_ fee associated to the respective payment method |
+| `min`                     | [`DecimalString`](#decimalstring)       | N        | minimum amount required to use this payment method.                                                                                                 |
+| `max`                     | [`DecimalString`](#decimalstring)       | N        | maximum amount allowed when using this payment method.                                                                                              |
+
+
+> [!IMPORTANT]
+> `estimatedSettlementTime` is used to provide a rough estimate for the expected latency between receiving a _payin_ and settling the respective _payout_.
```

Comment:
Wouldn't the estimated settlement time be a function of both the payin and the payout methods? Payin may be BTC which takes 10+ minutes when payout is debit which is instant

Reply:
Could maybe clarify: is this estimate supposed to be the full settlement of the exchange (starting from `Order` message being accepted -> `Close` message being sent), or is it JUST for the payout portion (which would be a `payout initiated/payin complete` OrderStatus -> `Close` message being sent)?

And does "receiving" a payin mean the PFI has been notified that there is a payin transaction (assuming the payin method is not stored balance), or does received == payin has _settled_?

Reply:
> Could maybe clarify: is this estimate supposed to be the full settlement of the exchange (starting from Order message being accepted -> Close message being sent), or is it JUST for the payout portion (which would be a payout initiated/payin complete OrderStatus -> Close message being sent)?

imo it should be _just_ for the payout portion because the PFI does not control when alice pays _in_. 

> And does "receiving" a payin mean the PFI has been notified that there is a payin transaction (assuming the payin method is not stored balance), or does received == payin has settled?

received == payin has settled. in a stored balance scenario, "payin received" can be considered as instant or moot

Reply:
It makes sense but IMO it's also relevant to the payin method. Agreed that the onus is on Alice for the payin method, but the PFI may have unique characteristics for their given payin methods. For example, consider BTC as a payin method, there is no standard rule as for how many blocks need to be mined for the payment to be considered final (even more relevant in Ethereum where reorgs are frequent), so each PFI could define their own set of rules for how long it takes for that tx to be considered final; Alice could submit a BTC tx and even see it in the blockchain but it may still not be considered settled by the PFI.